applications/vessel-history/src/features/search/SearchPlaceholders.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react'
+import cx from 'classnames'
+// TODO: replace this mocked useTranslation when i18n is implemented
+import { useTranslation } from 'utils/i18n'
+import vesselImage from 'assets/images/vessel@2x.png'
+import vesselNoResultsImage from 'assets/images/vessel-side@2x.png'
+import styles from './SearchPlaceholders.module.css'
+
+type SearchPlaceholderProps = {
+  className?: string
+  children?: React.ReactNode
+}
+
+function SearchPlaceholder({ children, className = '' }: SearchPlaceholderProps) {
+  return (
+    <div className={cx(styles.emptyState, className)}>
+      <div>{children}</div>
+    </div>
+  )
+}
+
+export function SearchNoResultsState({ className = '' }: SearchPlaceholderProps) {
+  const { t } = useTranslation()
+  return (
+    <SearchPlaceholder className={className}>
+      <img src={vesselNoResultsImage} alt="vessel" className={styles.noResultsImage} />
+      <p>
+        {t(
+          'search.noResults',
```

Comment:
only to know, what does mean this? you set the keyword of the translation but then you also add the text... so for what?

Reply:
oh, now I understand what you are doing seeing the last file